rl_games/common/player.py

```diff
@@ -43,14 +51,16 @@ def __init__(self, params):
         self.use_cuda = True
         self.batch_size = 1
         self.has_batch_dimension = False
-        self.has_central_value = self.config.get('central_value_config') is not None
+        self.has_central_value = self.config.get(
+            'central_value_config') is not None
         self.device_name = self.config.get('device_name', 'cuda')
         self.render_env = self.player_config.get('render', False)
         self.games_num = self.player_config.get('games_num', 2000)
         if 'deterministic' in self.player_config:
             self.is_deterministic = self.player_config['deterministic']
         else:
-            self.is_deterministic = self.player_config.get('determenistic', True)
+            self.is_deterministic = self.player_config.get(
```

Comment:
Misspelling error in config get, should be deterministic. @Denys88 maybe remove get and the default value to avoid error in the config? And check all the configs - it looks like some of them still have wrong "determenistic" spelling.